### ISSUE TITLE
LineChart: Wrong color of focus(nvd3 1.8.4)

### ISSUE THREAD

**Issue by fan-self** [novus/nvd3#1878]
The screenshot below are [examples of lineChart](https://github.com/novus/nvd3/archive/v1.8.4.zip) with some modifications:

Disable the color of each series:
 return [{
                area: true,
                values: sin,
                key: "Sine Wave",
                //color: "#ff7f0e",
                strokeWidth: 4,
                classed: 'dashed'
            }, {
                values: cos,
                key: "Cosine Wave"//,
                //color: "#2ca02c",
            }, {
                values: rand,
                key: "Random Points"//,
                //color: "#2222ff",
            }, {
                values: rand2,
                key: "Random Cosine",
                //color: "#667711",
                strokeWidth: 3.5
            }, {
                area: true,
                values: sin2,
                key: "Fill opacity",
                //color: "#EF9CFB",
                fillOpacity: .1
            }];

1. The initialization is correct.
![image](https://cloud.githubusercontent.com/assets/20783245/20779401/4cd397ee-b7af-11e6-9b14-9149a1dac2f6.png)

2. Disable some series, color of "Random Points" in focus is incorrect.
![image](https://cloud.githubusercontent.com/assets/20783245/20779418/678e1bb8-b7af-11e6-9226-1a6d8d214b45.png)

- The content of focus(nv.models.focus) is like this:
content
.width(availableWidth)
     .height(availableHeight)
     .color(data.map(function(d,i) {
               return d.color || color(d, i);
 	     	}).filter(function(d,i) { return !data[i].disabled; }));

- But the lineChart passed ‘!disabled dataset’  to focus, like this:
g.select('.nv-focusWrap')
.attr('transform', 'translate(0,' + (availableHeight + margin.bottom + focus.margin().top) + ')')
_.datum(data.filter(function(d) { return !d.disabled; }))_
.call(focus);

Suppose the italic statement should be removed.

**Comment by bgth** [comment]
Yes. I also checked. Its a bug.

**Comment by liquidpele** [comment]
Yup, seems to be a bug.  Please send a pull request with what you suggest as a fix :)

**Comment by emanuel1993** [comment]
I have this problem as well. This fix was merged in master, but not in 1.8.5.
Do you plan to release a new version soon? At the moment I have to use the master instead of the tag 1.8.5 for this fix to be available.

Thank you for your time!
